frontend/src/components/features/sidebar/sidebar.tsx

```diff
@@ -45,7 +44,8 @@ export function Sidebar() {
     React.useState(false);
 
   // TODO: Remove HIDE_LLM_SETTINGS check once released
-  const isSaas = HIDE_LLM_SETTINGS() && config?.APP_MODE === "saas";
+  const isSaas =
```

Comment:
should probably rename this var to sth like `should_hide_llm_settings` while we're here